kolibri/core/assets/src/vue/content-renderer/download-button.vue

```diff
@@ -58,20 +58,16 @@
       downloadMediaText() {
         return this.$tr('downloadMedia');
       },
+      downloadLink() {
+        const filePath = this.availableFiles[0].storage_url;
```

Comment:
We probably won't need to provide download links for the supplementary files, at least for now (I guess allowing download of subtitles could in theory be useful). But there might be multiple non-supplementary files, e.g. a low-res and a high-res video. We should probably show the list of all non-supplementary files and have download links for each. We can chat in person about what the UI might look like, if you want.